### DISCUSSION TITLE
Welcome to yahoofinancials Discussions!

### DISCUSSION THREAD

**Discussion by JECSand** [JECSand/yahoofinancials#117] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kondasantosh** [comment]
HI,
I am Santosh from India. About 3 to 4 months I have been digging deep in python applications in financial markets and I having fun spending day and night learning about these thinks and have applied these thing in real life and gain some profits. I would like to contribute and Collab with your work. My current ambition are to get 5cr india ruppee which would be enough for me to place them in low risk gov securities so that I could achieve some time, money and place freedom. My life till now as a person have been a total introvert and given opportunity stayed in house because my social standing as a son of single mother. I want to change that and  having a fulling time job in software domain and learning these things in parallel will give me awareness to grab future opportunity. Till  last year for some reason I feel like I was in trans state or depression I have some improvement near that area and exactly because of that reason I am able to discuss it here.
